eng/SourceBuild.props

```diff
@@ -62,11 +62,6 @@
 
       <IntermediateNupkgArtifactFile Include="$(CurrentRepoSourceBuildArtifactsPackagesDir)Shipping\*Microsoft.DotNet.ILCompiler.*.nupkg" Category="ILCompiler" />
 
-      <IntermediateNupkgArtifactFile
```

Comment:
I don't think this is going to achieve what you want.  IIRC, all packages that are not included in supplemental packages get included in the default master package.

Reply:
Yeah, Matt's working on an alternative. Closing this one.